### ISSUE TITLE
Authentication not getting passed to embedded envelope requests

### ISSUE THREAD

**Issue by rachelbaker** [WP-API/WP-API#837]
To reproduce:

1. Send an authenticated request to `wp-json/wp/posts/{ID}?_envelope&_embed`
2. Note the permission error in the embedded author response.  

```json
"_embedded": {
            "author": [
                [
                    {
                        "code": "json_user_cannot_list",
                        "message": "Sorry, you are not allowed to view this user.",
                        "data": {
                            "status": 403
                        }
                    }
                ]
            ],
```

**Comment by danielbachhuber** [comment]
#838 will fix this, but #839 is really what we need to solve.

**Comment by rmccue** [comment]
To address the title:

> Authentication not getting passed to embedded envelope requests

There's no way that *this* can happen, as both just use WP's built-in authentication system, not the API routing/parameter system.

This is likely just #838, so closing.
